backend/vkt/src/main/java/fi/oph/vkt/config/Constants.java

```diff
@@ -9,4 +9,8 @@ public class Constants {
 
   // Daily at 9:00
   public static final String DELETE_CANCELED_UNFINISHED_ENROLLMENTS_CRON = "0 0 9 * * *";
+  // Daily at 9:30
+  public static final String DELETE_EXPIRED_RESERVATIONS_CRON = "0 30 9 * * *";
+  // Daily at 10:00
+  public static final String DELETE_OBSOLETE_PERSONS_CRON = "0 0 10 * * *";
```

Comment:
Ei varmaan käytännössä ole merkitystä kun VKT:ssä ei lie samanlaista ruuhkaa mitä YKI:ssä, mut oiskohan silti parempi ajastaa nämä aamuyöhön. 🤔 Tyyliin klo 6 tai 7 aikoihin.

Reply:
Sovellukset ei oo untuvalla ajossa ilmeisesti klo 19 ja ehkä aamulla klo 7 välillä? Joskus aiemmin tais akr/otr päässä olla noille skedulereille ajot merkattu tapahtuvan aamuyöllä ja eivät näin untuvalla koskaan ajautuneet. Päätettiin sen myötä siirtää ne aamuun klo 9 eteenpäin. Tän syyn vois sinänsä laittaa kommentilla tänne Constants alle tai sit muuten dokumentaatioon.

Reply:
Aa, okei. 👍